modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb

```diff
@@ -8,7 +8,7 @@ module QuestionnaireResponse
       class Factory
```

Comment:
Please add class-level and public method documentation via [YARDoc](https://yardoc.org/guides/index.html)

Reply:
Done!